src/index.html

```diff
@@ -13,20 +13,32 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="style.scss"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      class="form"
+    >
+      <input
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles“"
```

Comment:
The placeholder text 'Try “Los Angeles“' is repeated in both input fields. It's better to have unique placeholder text for each input field to guide the user on what to input.